packages/sindarian-ui/src/utilities.css

```
@@ -0,0 +1,22 @@
+/* Scrollbar — thin translucent thumb for scrollable layout containers */
+@utility scrollbar-thin-translucent {
+  scrollbar-width: thin;
+  scrollbar-color: var(--color-shadcn-300) transparent;
+
+  &::-webkit-scrollbar {
```

Stylelint (17.4.0)

[error] Unexpected missing scoping root (nesting-selector-no-missing-scoping-root)

(nesting-selector-no-missing-scoping-root)